libraries/AP_SerialManager/AP_SerialManager.cpp

```diff
@@ -168,7 +172,18 @@ void AP_SerialManager::init()
     if (state[0].uart == nullptr) {
         init_console();
     }
-    
+
+#if HAL_WITH_UAVCAN
+    for (uint8_t can_mgr = 0; can_mgr < MAX_NUMBER_OF_CAN_DRIVERS; can_mgr++) {
+        AP_UAVCAN *ap_uavcan = AP_UAVCAN::get_uavcan(can_mgr);
+        const uint8_t state_index = 5 + can_mgr;
```

Comment:
that '5' constant will bite us later, please make it a define up near the other stuff for uarts so we don't miss it when we add uartG